docs/design-docs/02-custom-root-reconciler-clusterrole.md

```diff
@@ -18,65 +24,130 @@ This proposal introduces a mechanism to customize this behavior.
 If one wants to allow deploying resources from a source where one doesn't have full
```

Comment:
I know you didn't write this, but this sentence is pretty hard to read. I would get rid of the "one" and rephrase with "you" and maybe reorganize this section as a list of use cases. This use case is for a platform admin wanting to allow tenants to use RootSyncs with reduced permissions. 

The use case for RepoSync is to follow least privilege to only give the reconciler the permissions it needs, to limit the blast area if the source of truth is compromised or a developer mistake is made. For example, to disallow making service accounts and RBAC.

Reply:
Done